system/CLI/Console.php

```diff
@@ -31,6 +32,10 @@ class Console
      */
     public function run()
     {
+        // Create CLIRequest
+        $appConfig = config(App::class);
```

Comment:
```suggestion
        $appConfig = config('App');
```

Reply:
Why? `App::class` better

Reply:
You're right, I forgot I was reviewing CI PR.